core/src/main/java/org/hisp/dhis/android/core/category/CategoryOptionComboCategoryLinkStoreImpl.java

```diff
@@ -37,14 +69,51 @@ public long insert(@NonNull CategoryOptionComboCategoryLinkModel entity) {
         return executeInsert();
     }
 
+    @Override
+    public int delete() {
+        return databaseAdapter.delete(CategoryOptionComboCategoryLinkModel.TABLE);
+    }
+
+    @Override
+    public boolean delete(@NonNull CategoryOptionComboCategoryLinkModel element) {
+        validate(element);
+
+        bind(deleteStatement, element);
+
+        return execute(deleteStatement);
+    }
+
+    @Override
+    public boolean update(
```

Comment:
To update one row only is necessary one model, you are pass as argument new and old, can you refactor this method?

Reply:
Please, take a look at the example on CategoryCategoryOptionLink, the same applies here.